src/main/java/core/basesyntax/service/TransactionStrategyImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service;
+
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.transaction.TransactionHandler;
+import java.util.HashMap;
+
+public class TransactionStrategyImpl implements TransactionStrategy {
+    private HashMap<FruitTransaction.Operation, TransactionHandler> operationMap;
```

Comment:
Use the interface 'Map' instead of its implementation 'HashMap' for the variable type.